File: Uri 1013 - O Maior/Main.py
A, B, C= [int(x) for x in input().split()]
MaiorAB = (A + B + abs(A - B)) / 2
MaiorAB = (C + MaiorAB + abs(MaiorAB - C)) / 2
print("%d eh o maior"%MaiorAB)

File: Uri 1013 - O Maior/Main.c
#include <stdio.h>

int main() {
 int A, B, C, MaiorAB;
scanf("%d %d %d", &A, &B, &C);
MaiorAB =(A+B+abs(A-B))/2;
MaiorAB=(C+MaiorAB+abs(MaiorAB-C))/2;
printf("%d eh o maior\n",MaiorAB);
    return 0;
}